src/Common/tests/System/Collections/IEnumerable.Generic.Tests.cs

```diff
@@ -730,5 +732,38 @@ public void IEnumerable_Generic_Enumerator_Reset()
         }
 
         #endregion
+
+        #region Serialization
+        #if netstandard17
```

Comment:
For the test cs files we would still like to split these netstandard17 only test methods out into their own files.